public-ui/src/app/data-browser/concept-charts/concept-charts.component.html

```diff
@@ -6,12 +6,117 @@
 
 <div *ngIf="unitNames.length > 1" class="col-md">
   <button *ngFor="let u of unitNames.slice(0,8)" [ngClass]="{active:selectedUnit === u}" (click)="showMeasurementGenderHistogram(u)" class="btn btn-link">{{u}} </button>
+  <clr-tooltip>
+    <clr-icon clrTooltipTrigger shape="info-standard" class="info-icon"></clr-icon>
+    <clr-tooltip-content clrPosition=right
+                         clrSize="lg" *clrIfOpen>
+      <div class="tooltip-label">Toggle here to view unit specific histograms</div>
+    </clr-tooltip-content>
+  </clr-tooltip>
 </div>
 
 <div *ngIf="!loading()" class="row concept-charts chart">
   <!-- show measurements binned by gender if we want -->
   <!-- Loop through the genders we have measurement results for  for showing the graph
    This looping variable is set if showMeasurements is passed in to the component-->
+  <ng-container *ngIf="(showGender || showGenderIdentity) && showAge && (showSources && sourceConcepts.length > 0) && !showMeasurementGenderBins">
```

Comment:
This is a pretty complex predicate. I would recommend putting this int the component.ts as a method and calling that here, e.g. `showX` (not sure what X is here.

Reply:
Moved it into a method in .ts